### ISSUE TITLE
Default install path on Windows should be Program Files or Program Files (x86)

### ISSUE THREAD

**Issue by BoffinbraiN** [telegramdesktop/tdesktop#3903]
### Steps to reproduce
1.  Start installing TDesktop on a new Windows install

### Expected behaviour
The installer should offer to install at one of...

- C:\Program Files
- C:\Program Files (x86)

...where C: is the system drive.

### Actual behaviour
It tries to install to C:\Users\YourUsername\AppData\Roaming

### Configuration
Windows 7 SP1

**Version of Telegram Desktop:**
1.1.23

The semantics of the AppData\Roaming directory is that it should be used for user preferences, not for installing entire programs. AppData\Local is intended for stuff that's kept on that physical machine and does not roam, such as caches. It's done that way to keep roaming profiles light when using a profile on multiple computers connected to a Windows domain.

A properly configured Windows application should store its executables, DLLs and dependencies under one of the 'Program Files' folders, and user-specific data in AppData as described above.

As it currently stands, if a computer has two or more users that use Telegram, the software is duplicated in each user's AppData folders and wastes disk space.

Nearly all Windows programs obey these guidelines, so Telegram should try to do the same.

See also: https://superuser.com/questions/21458/why-are-there-directories-called-local-locallow-and-roaming-under-users-user

**Comment by stek29** [comment]
#2355

**Comment by stek29** [comment]
TL;DR: If you start installer as admin then it should support installation to program files (iirc)

**Comment by BoffinbraiN** [comment]
How are users supposed to discover that? No wonder I've duplicated a previous request.
There should be an option during installation to install for all users or the current user. At that point, the installer should ask for admin privileges.

**Comment by stek29** [comment]
@BoffinbraiN Then there will be people creating "help updating broken" issues. So updater should be started with admin privileges too somehow. 

>  At that point, the installer should ask for admin privileges.

iirc it has to be restarted with admin privileges to work properly.

**Comment by pleappleappleap** [comment]
I just tried installing after starting with admin privileges and it still didn't Do the Right Thing(tm).  I was, however, using Avecto to gain admin privileges.
